ui/tests/test-helper.js

```diff
@@ -7,6 +7,13 @@ import { start } from 'ember-qunit';
 import { setup as setupA11yTesting } from './helpers/a11y';
 import './helpers/flash-message';
 
+Object.defineProperty(navigator, 'platform', {
+  value: 'Macintosh',
+  configurable: true,
+  enumerable: true,
+  writable: false,
+});
```

Comment:
This is an ultra-hack to get around the issue mentioned here: https://github.com/hashicorp/waypoint/pull/2901#issuecomment-1012275634

Reply:
If it works (CI passes) then I can document why this is here.

Reply:
Looks like it's good. let's add a comment and 🚢